zerver/views/user_settings.py

```diff
@@ -82,6 +118,16 @@ def json_change_settings(request, user_profile,
         request.session.save()
 
     result = {}
+    new_email = email.strip()
+    if user_profile.email != email and new_email != '':
+        error, skipped = validate_email(user_profile, new_email)
+        if error or skipped:
+            return json_error(error or skipped)
+
+        do_start_email_change_process(user_profile, new_email)
+        result['account.email'] = _('We have sent you an email on your '
```

Comment:
We should probably just make this `result['email_change']` since we don't really use that `.` style elsewhere.